lib/models/wallet_connect/approve_token_transfer_result.dart

```diff
@@ -1,16 +1,25 @@
 /// Results for approving token transfer
 class ApproveTokenTransferResult {
   final int? tick;
+  final String? transactionId;
   final String? errorMessage;
   final int? errorCode;
   ApproveTokenTransferResult(
-      {required this.tick, this.errorCode, this.errorMessage});
+      {required this.tick,
+      required this.transactionId,
+      this.errorCode,
+      this.errorMessage});
 
   ///Important, always provide a toJson otherwise WC serilization will fail
   Map<String, dynamic> toJson() {
     if (errorMessage == null && errorCode == null) {
-      return {'tick': tick};
+      return {'tick': tick, 'transactionId': transactionId};
     }
-    return {'tick': tick, 'errorMessage': errorMessage, 'errorCode': errorCode};
+    return {
+      'tick': tick,
```

Comment:
On error there is no need to send tick and transactionId as null.

Reply:
Fixed with 3f37417